test/data/OwnersSpec.scala

```diff
@@ -5,14 +5,6 @@ import org.specs2.mutable._
 
 class OwnersSpec extends Specification {
 
-  "A guardian stack" should {
-    "have only one owner" in {
```

Comment:
You're right it's not enforced. I figured that because it was hardcoded it would be enough.
I'll add a test that checks the hardcoded values don't contain twice the same ID.